website/docs/d/supervisor_namespace.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: "vcfa"
+page_title: "VMware Cloud Foundation Automation: vcfa_supervisor_namespace"
+sidebar_current: "docs-data-source-vcfa-supervisor-namespace"
+description: |-
+  Provides a VMware Cloud Foundation Automation Supervisor Namespace data source.
+---
+
+# vcfa\_supervisor\_namespace
+
+Provides a VMware Cloud Foundation Automation Supervisor Namespace data source.
+
+## Example Usage
+
+```hcl
+data "vcfa_supervisor_namespace" "supervisor_namespace" {
+  name         = "demo-supervisor-namespace"
+  project_name = "default-project"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Supervisor Namespace
+* `project_name` - (Required) The name of the Project where the Supervisor Namespace belongs to
+
+## Attribute Reference
+
+- `class_name` - The name of the Supervisor Namespace Class
+- `description` - Description
+- `phase` - Phase of the Supervisor Namespace
+- `ready` - Whether the Supervisor Namespace is in a ready status or not
+- `region_name` - Name of the Region
+- `storage_classes` - A set of Supervisor Namespace Storage Classes. See [Storage Classes](#storage-classes) section for details
+- `storage_classes_initial_class_config_overrides` - A set of Supervisor Namespace Storage Classes Initial Class Config Overrides. See [Storage Classes Initial Class Config Overridess](#storage-classes-initial-class-config-overrides) section for details
+- `vpc_name` - Name of the VPC
+- `vm_classes` - A set of Supervisor Namespace VM Classes. See [VM Classes](#vm-classes) section for details
+- `zones` - A set of Supervisor Namespace Zones. See [Zones](#zones) section for details
+- `zones_initial_class_config_overrides` - A set of Supervisor Namespace Zones Initial Class Config Overrides. See [Zones Initial Class Config Overrides](#zones-initial-class-config-overrides) section for details
+
+<a id="storage-classes"></a>
+## Storage Classes
+
+The `storage_classes` is a set of metadata entries that have the following structure:
```

Comment:
I see "metadata entries" in every block. Is it because it's metadata inside K8s YAML or is it a typo?

Reply:
Same as above.

Reply:
Done.